templates/maas/features.html

```diff
@@ -0,0 +1,526 @@
+          <a href="/maas/docs/how-to-enhance-maas-security#p-9102-use-tls-termination-maas-33"
+            onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'Network management' });"
+            aria-label="Link to install and configure secure sockets layer on MAAS documentation"><abbr
+              title="Secure sockets layer">SSL</abbr></a>
+        </li>
+        <li>
+          <a href="/maas/docs/how-to-manage-networks"
+            onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'Network management' });"
+            aria-label="Link to install and configure spanning tree protocol on MAAS documentation"><abbr
+              title="Spanning tree protocol">STP</abbr></a>
+        </li>
+        <li>
+          <a href="/maas/docs/how-to-manage-networks"
+            onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'Network management' });"
+            aria-label="Link to install and configure network proxy on MAAS documentation">Proxy</a>
+        </li>
+      </ul>
+    {%- endif -%}
+  {%- endcall -%}
+  {% call(slot) vf_equal_heights(
+    title_text="Create and manage custom LXD VMs",
+    image_aspect_ratio_small="auto",
+    image_aspect_ratio_medium="auto",
+    image_aspect_ratio_large="auto",
+    items=[
+      {
+        "title_text": 'Graphical overview of your KVM host’s resources',
+        "image_html": lightbox_image(
+          img="https://assets.ubuntu.com/v1/87982747-graphical_overview_1.jpg",
+          description="pod details",
+        ),
+        "description_html": '<p>MAAS shows your KVM pod’s CPU cores and RAM, as well as the free space in storage pools.</p>
+          <p>Manage and visualize overcommit ratios.</p>',
+      },
+      {
+        "title_text": 'Choose storage from multiple pools',
+        "image_html": lightbox_image(
+          img="https://assets.ubuntu.com/v1/8c09b579-choose_storage_1.jpg",
+          description="storage details",
+        ),
+        "description_html": '<p>Assign one or more volumes from the default and other available pools.</p>
+          <p>View the total size of your requests and the free space remaining.</p>',
+      },
+      {
+        "title_text": 'Customize a VM’s network interfaces',
+        "image_html": lightbox_image(
+          img="https://assets.ubuntu.com/v1/19d80432-customize_1.jpg",
+          description="network interfaces",
+        ),
+        "description_html": '<p>Create software defined networking as complex as you need.</p>
+          <p>Assign interfaces to spaces for Juju models, browse subnets by fabric and VLAN, or simply type an IP number.</p>',
+      },
+    ]
+  ) %}
+    {% if slot == "description" %}
+      <p>
+        Kernel-based Virtual Machine (KVM) is the core Linux kernel technology that enables hardware virtualization. LXD
```

Comment:
"LXD" is bolded in the [copydoc](https://docs.google.com/document/d/1KcYwNK0pJyiDVEDq4vTpv6BX_rpSDnxUxtAvAnsRKyw/edit?tab=t.0#heading=h.ulx3j2uqto0a)
```suggestion
        Kernel-based Virtual Machine (KVM) is the core Linux kernel technology that enables hardware virtualization. <strong>LXD</strong>
```

Reply:
Thanks for flagging @jmuzina, I think Mattea dropped this from the design as we don't typically bold text within paragraphs. The design was approved by the PM and I saw this PR was merged anyway so happy to go with this version. I've edited the formatting in the copydoc for consistency